src/import-utils.ts

```diff
@@ -94,6 +117,7 @@ export async function inlineHtmlImport(
   }
 
   const importDoc = parse5.parseFragment(importSource);
+  debaseDocument(resolvedImportUrl, importDoc);
```

Comment:
Well, we don't need to call `rewriteImportedUrls` on our bundle entrypoint doc, but we *do* need to call it when we `inlineHtmlImport`.  However, we need to call `rewriteDocumentToEmulateBaseTag` (formerly `debaseDocument`) on all documents.  That's why they're split up.